pkgs/servers/mail/public-inbox/default.nix

```diff
@@ -141,7 +163,7 @@ buildPerlPackage rec {
           git
           /* for InlineC */
           gnumake
-          stdenv.cc.cc
+          stdenv.cc
```

Comment:
This doesn't seem to have been enough to fix InlineC — it now complains about crypt.h missing, so presumably we need libxcrypt exposed as well.